images/base/files/etc/systemd/system/kubelet.service.d/10-kubeadm.conf

```diff
@@ -9,4 +9,4 @@ EnvironmentFile=-/var/lib/kubelet/kubeadm-flags.env
 # the .NodeRegistration.KubeletExtraArgs object in the configuration files instead. KUBELET_EXTRA_ARGS should be sourced from this file.
 EnvironmentFile=-/etc/default/kubelet
 ExecStart=
-ExecStart=/usr/bin/kubelet $KUBELET_KUBECONFIG_ARGS $KUBELET_CONFIG_ARGS $KUBELET_KUBEADM_ARGS $KUBELET_EXTRA_ARGS
\ No newline at end of file
+ExecStart=/usr/bin/kubelet $KUBELET_KUBECONFIG_ARGS $KUBELET_CONFIG_ARGS $KUBELET_KUBEADM_ARGS $KUBELET_EXTRA_ARGS --cgroup-root=/kubelet
```

Comment:
for a follow-up: we should inject this with kubelet config, flags are basically deprecated.

Reply:
there are tricker per-node kubelet flags injected right now (providerID) that will require a larger overhaul (kubeadm doesn't support per-node kubelet config, only flags...), so we can punt this until then.

and when we do that, the flag here should override the config, which would make sense given that the image uses a fixed location for now.

so this should work quite well.